int32 reward
geometry_msgs/Point target_position